src/valora-dapp-list.json

```diff
@@ -266,44 +266,44 @@
       "categoryId": "lend-borrow-earn",
       "categories": ["earn"],
       "url": "https://app.stcelo.xyz/",
-      "listOnAndroid": true,
-      "listOnIos": true
+      "listOnAndroid": false,
+      "listOnIos": false
```

Comment:
seems to be working as of today

Reply:
The page is not loading for me:
https://app.stcelo.xyz/

But I'm reverting it